packages/@react-spectrum/numberfield/src/NumberField.tsx

```diff
@@ -35,11 +36,18 @@ function NumberField(props: SpectrumNumberFieldProps, ref: FocusableRef<HTMLElem
   let {
     isQuiet,
     isDisabled,
-    hideStepper
+    hideStepper,
+    formatOptions
   } = props;
 
   let {styleProps: style} = useStyleProps(props);
-  let state = useNumberFieldState(props);
+
+  let {locale} = useLocale();
+  let [currentNumeralSystem, setCurrentNumeralSystem] = useState<NumeralSystem | undefined>();
```

Comment:
Should this be in stately? Seems weird to pass it around as a prop, especially the setter.

Reply:
So reason for this was the NumeralSystem type from react-aria, but following the comment below, I'll just move that type into locale.d.ts